app/_includes/plugins/kubernetes_consumer.html

```diff
@@ -0,0 +1,29 @@
+{% if include.hub_example.auth_plugin? %}
+
+As this is an auth plugin, we need to create a credential for the plugin. Create a secret with the following command:
+
+```bash
+kubectl create secret generic alice-{{ include.hub_example.plugin_name }} \
+  --from-literal=kongCredType={{ include.hub_example.plugin_name }}{%- for field in include.hub_example.auth_fields %} \
+  --from-literal={{ field[0] }}={{ field[1] }}{%- endfor %}
+```
+
+Finally, we create a consumer that will use this credential:
+
+```bash
+echo "apiVersion: configuration.konghq.com/v1
+kind: KongConsumer
+metadata:
+  name: alice
```

Comment:
@mheap I love the name `alice` but this is what we need to follow: https://developers.google.com/style/examples